### DISCUSSION TITLE
Removing replica instance manager and keeping engine instance manager

### DISCUSSION THREAD

**Discussion by thecodeassassin** [longhorn/longhorn#6836] [General]
Hi all,

I have Longhorn deployed with helm, I'd like to have the ability to mount volumes on any nodes but to only host replicas on nodes with 10g uplinks.

Is it possible to only run the engine instance manager on every node and not the replica instance manager? I couldn't find any value in the Helm chart for this. Of course I could manually patch the deamonset but this seems really hacky.

**Comment by c3y1huang** [comment]
[Tip: Set Longhorn To Only Use Storage On A Specific Set Of Nodes](https://longhorn.io/kb/tip-only-use-storage-on-a-set-of-nodes/) might help.

**Reply by thecodeassassin** [reply]
I know that this is a thing, I could patch the daemonset manually but it would be nice if this could be controlled by the helm chart. Should I maybe submit a PR for this feature?
